about.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/about.css">
+    <script src="/components/header.js"></script>
+    <script src="/components/footer.js"></script>
+    <title>About</title>
+</head>
+<body>
+    <header-component></header-component>
+    <main class="main-section">
```

Comment:
- Kindly add a way to navigate between the home and about page in the mobile version.